etc/testing/deploy-manifests/golden/amazon-deploy-manifest.json

```diff
@@ -515,7 +515,7 @@
         "containers": [
           {
             "name": "pachd",
-            "image": "pachyderm/pachd:1.13.2",
+            "image": "pachyderm/pachd:1.13.2-CIbuild",
```

Comment:
The -CIbuild may be fine here, but when we do a release this will change to 1.13.3. I wonder how that would break. IIRC, -CIbuild was needed for testfaster to tag/pull a known image in its layers of cache.

Can you try one more thing... remove the -CIbuild from the top level Makefile? Let it build without VERSION_ADDITIONAL. If that works, that might be the right answer instead of changing the golden manifest.